src/store/flow/index.ts

```diff
@@ -19,21 +19,6 @@ export interface IFlowsState {
   selectedBlocks: IBlock['uuid'][],
 }
 
-export const stateFactory = (): IFlowsState => ({
```

Comment:
had to move this into `src/store/flow/flow.ts` to avoid some kind of lexical issue, because I'm using it there (see this line https://github.com/FLOIP/flow-builder/pull/458/files#diff-eea673c657b7edcdbf00ab176fcc8ec0a84d21e8a8dfa9ff8a014c7510a144a0R115) . I forgot to take a snapshot of the error.